pkg/trace/api/api.go

```diff
@@ -69,7 +71,9 @@ type HTTPReceiver struct {
 
 	maxRequestBodyLength int64
 	debug                bool
+	refuse               int64 // when set to 1 agent will refuse payloads
```

Comment:
I see that you are using it as an atomic boolean. Maybe wrap it in a specific type so that its usage/bahavior is clear? (first time I read it, I was expecting `refuse` allow more values). 
Example: https://github.com/tevino/abool/blob/master/bool.go

Reply:
I'd say we ignore this for the sake of this iteration. We will absorb this behavior into the presampler in a follow up PR.